packages/dds/tree/src/tableSchema.ts

```diff
@@ -326,23 +320,12 @@ export namespace TableSchema {
 		propsSchema: TPropsSchema,
 	): ReturnType<typeof createRowInternal<TInputScope, TCellSchema, TPropsSchema>>;
 	/** `createRow` implementation */
-	export function createRow<
-		const TInputScope extends string | undefined,
-		const TCellSchema extends ImplicitAllowedTypes,
-		const TPropsSchema extends ImplicitFieldSchema = ImplicitFieldSchema,
-	>(
-		inputSchemaFactory: SchemaFactoryAlpha<TInputScope>,
-		cellSchema: TCellSchema,
-		propsSchema?: TPropsSchema,
+	export function createRow(
+		inputSchemaFactory: SchemaFactoryAlpha,
+		cellSchema: ImplicitAllowedTypes,
+		propsSchema: ImplicitFieldSchema = inputSchemaFactory.optional(inputSchemaFactory.null),
```

Comment:
I suspect is not a good idea, but its worth noting its actually possibly to express a field that must always be undefined. `SchemaFactory.optional([])`

Also, for creating field schema, might as well just use the static methods instead of inputSchemaFactory since they are stateless.
```suggestion
		propsSchema: ImplicitFieldSchema = SchemaFactory.optional(SchemaFactory.null),
```